src/actions/summaryActions.js

```diff
@@ -0,0 +1,33 @@
+import {
+  SUMMARY_FETCHING,
+  SUMMARY_RECEIVED,
+} from './actionTypes'
+import api from '../util/api'
+
+export const fetchingSummary = () => ({
+  type: SUMMARY_FETCHING,
+})
+
+export const receivedSummary = summaries => ({
+  type: SUMMARY_RECEIVED,
+  summaries,
+})
+
+// TODO: rename this. fetchSummaries isn't very descriptive
+// but something like fetchPlaceAndNationalSummaries is too long
+export const fetchSummaries = params => dispatch => {
```

Comment:
@brendansudol I'm not comfortable with this naming, but we can change it when we come up with a better one